README.rst

```diff
@@ -60,6 +60,11 @@ You can select a different format using the ``--docformat`` option.
 What's New?
 ~~~~~~~~~~~
 
+pydoctor dev
+^^^^^^^^^^^^
+
+* Removed the ``--html-write-function-pages`` option. Please use the generated Intersphinx inventory (objects.inv) for deep-linking your documentation.
```

Comment:
No need for the "please" part :)

There was a guide (by Google or maybe Gnome) in which it was described how to write user facing text  / messages.
I remember that one of the now as to not use "Please"

```suggestion
* Removed the ``--html-write-function-pages`` option. As a replacement, you can use the generated Intersphinx inventory (objects.inv) for deep-linking your documentation.
```